cluster-scope/base/operators.coreos.com/subscriptions/vertical-pod-autoscaler/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: cs6620-fall21-deployverticalpod
```

Comment:
recommended namespace is `openshift-vertical-pod-autoscaler`, 

It seems to me the VPA ought to be deployed there, and project-admins are then able to deploy the custom resource in their respective projects, despite VPA being installed to a 'namespace' 

@4n4nd @tumido do you know?